pkg/providers/types/provider/provider_types.go

```diff
@@ -0,0 +1,8 @@
+package provider
+
+type ProviderType string
+
+const (
+	ProviderTypeCF     ProviderType = "cf"
```

Comment:
Whenever possible, I prefer to avoid acronyms such as here since it can lead to confusion. What do you think of renaming this to `cloud foundry`?